src/insert.rs

```diff
@@ -28,19 +28,27 @@ const MIN_CHUNK_SIZE: usize = BUFFER_SIZE - 1024; // slightly less to avoid extr
 /// Rows are being sent progressively to spread network load.
 #[must_use]
 pub struct Insert<T> {
+    client: Client,
+    table: String,
+    fields: String,
```

Comment:
It's slightly better (for reading) to store formatted `sql` here, not only `fields`. Thus, both `table` and `fields` will be replaced with one `sql`